src/components/Tag/native/Tag.js

```diff
@@ -3,37 +3,41 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import Text from '../../Text/native';
 
+const styles = {
+  background(props) {
+    if (props.kind === 'filled') return props.theme.color[props.color];
+    return props.theme.color[`${props.color}Lighter`];
+  },
+  padding(props) {
+    if (props.size === 'small') return props.theme.px([0.15, 0.5, 0.25, 0.5]);
```

Comment:
Vertical padding not consistent ?

Reply:
Good catch! 

@kashishgrover could you look into this and make sure it's as per our guidelines?